index.html

```diff
@@ -162,7 +162,7 @@ <h5 class="modal-title" id="sponsorModalLabel">Sponsor a Family</h5>
   
 
  <!--Our source JS--> 
- <script src="/assets/js/script.js"></script>  
```

Comment:
The script referencing the script.js file is also referenced at top in line 12. We should only keep the script elements at the top, or only at the bottom of the HTML, under Our source JS

Reply:
We will change this in next push.  I took Chhaya as a reviewer - do you merge this now as the reviewer? I need to go back to my notes on how this process works.